src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -83,7 +83,7 @@ class HaAutomationPicker extends LitElement {
               ${automation.attributes.last_triggered
                 ? formatDateTime(
                     new Date(automation.attributes.last_triggered),
-                    this.hass.language
+                    this.hass.locale
```

Comment:
Because we use `this.hass.locale` here, we should use that as the memorize function parameter instead of `this.hass.language`.